.github/workflows/format-check.yml

```diff
@@ -0,0 +1,37 @@
+name: format-check
+on:
+  push:
+    branches:
+      - 'main'
+      - /^release-.*$/
+    tags: ['*']
```

Comment:
we only fail this workflow when reviewdog runs so there's not much sense in running hte workflow if the reviewdog step isn't run...

Reply:
we do run reviewdog on push though?

Reply:
nope, there's an `if: github.event_name == 'pull_request'` there

Reply:
ohh I totally missed that

Reply:
```suggestion
```